src/main/scala/com/raphtory/core/components/querymanager/QueryHandler.scala

```diff
@@ -126,6 +127,7 @@ abstract class QueryHandler(
     }
     else {
       readyCount += 1
+      QueryTelemetry.readyCount.inc()
```

Comment:
If we are tracking ready counts we probably want some more information on what they are reading for, can probably reset them for each step. This may not tell us too much at the moment.